presto-native-execution/presto_cpp/main/PrestoServerOperations.h

```diff
@@ -23,13 +23,14 @@ class ResponseHandler;
 
 namespace facebook::presto {
 
+class PrestoServer;
 struct ServerOperation;
 
 /// Static class implements Presto Server Operations.
 class PrestoServerOperations {
  public:
-  PrestoServerOperations(TaskManager* const taskManager)
-      : taskManager_(taskManager) {}
+  PrestoServerOperations(TaskManager* const taskManager, PrestoServer* server)
```

Comment:
Can we access task manager from Presto server by providing an accessor in PrestoServer? Then we just need to pass PrestoServer here?

Reply:
We can do this, however that would break some unit tests, where we have task manager, but not the server.